lib/throttle.js

```diff
@@ -25,6 +25,21 @@ module.exports = (api) => {
     };
   };
 
+  api.metasync.debounce = (
+    timeout,
+    fn,
+    args
+  ) => {
+    let timer = null;
+    return () => {
+      clearTimeout(timer);
```

Comment:
There is nothing wrong doing it. I did follow `throttle` code style while writing `debounce`.